pallets/xyk/src/lib.rs

```diff
@@ -34,7 +34,8 @@ use frame_support::sp_runtime::{
 };
 use frame_support::{dispatch::DispatchResult, ensure, traits::Get, transactional};
 use frame_system::ensure_signed;
-use primitives::{asset::AssetPair, fee, traits::AMM, AssetId, Balance, Price, MAX_IN_RATIO, MAX_OUT_RATIO};
+use primitives::{asset::AssetPair, fee, traits::AMM, AssetId, Balance, Price};
+use common_runtime::constants::chain::{MAX_IN_RATIO, MAX_OUT_RATIO};
```

Comment:
is this a wanted change?

Reply:
I think it makes sense to store common constants in `common_runtime`. There is also an option to create a new submodule for constants in `primitives` module.